docs/application/dotnet/guides/app-management/data-control.md

```diff
@@ -34,51 +34,58 @@ The main features of the Tizen.Applications.DataControl namespace include:
 
 The data control use cases involve 2 applications. Each application plays a different role: one as the consumer, the other as the provider.
 
-To enable your application to use the data control functionality:
+To enable your application to use the data control functionality, you have to include [Tizen.Applications.DataControl](https://samsung.github.io/TizenFX/latest/api/Tizen.Applications.DataControl.html) namespace.
+```
+using Tizen.Applications.DataControl;
+```
+
+#### Consumer
+Consumer application needs to request specific permission by adding the following privileges to the `tizen-manifest.xml` file:
+```
+<privileges>
+    <privilege>http://tizen.org/privilege/datasharing</privilege>
+    <privilege>http://tizen.org/privilege/appmanager.launch</privilege>
+</privileges>
+```
+
+#### Provider
+
+For your privider, you need to include a few things in `tizen-manifest.xml`. If you are using Visual Studio, double-click `tizen-manifest.xml`, and in the manifest editor, go to `Advanced > Data Control`, and click `Add` to add the provider detail. Add the `Read` and `Write` access rights to both `SQL` and `Map` types, as needed.
```

Comment:
provider

Reply:
go to `Advanced > Data Control`, and click `Add` to add the provider detail. 

@mijin85cho, the above code tags should it be changed to bold, need your opinion. In other files for menu options that is what we have followed.

File names are in code tag here, that I am leaving as is.

Reply:
Yes, To the UI elements, please apply bold font.

Reply:
@mijin85cho, thanks.

@MathewP1, here is the suggested change:

For your provider, you need to include a few things in `tizen-manifest.xml`. If you are using Visual Studio, double-click `tizen-manifest.xml`, and in the manifest editor, go to **Advanced** > **Data Control**, and click **Add** to add the provider detail. Add the `Read` and `Write` access rights to both `SQL` and `Map` types, as needed.

Reply:
My bad, I've fixed the style.